strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/configuration/ProxyConfiguration.java

```diff
@@ -39,8 +39,7 @@
      */
     @XmlAttribute
     private String type;
-
-    @XmlElement(name = "host")
```

Comment:
Yes. There was an error related with ambiguous host xml element. See above in this file, there already exists

> @XmlAttribute
    private String host;